app/viewmodels/ResultsPageSummary.scala

```diff
@@ -46,19 +49,29 @@ class ResultsPageSummary(from: LocalDate, to: LocalDate)(implicit messages: Mess
         content = span(messages("cf.cash-account.detail.csv")),
         visuallyHiddenText = Some(messages("cf.cash-account.detail.csv-definition"))
       )))
+
     )
   }
 
-  private def rowResult(dates: CashTransactionDates): String = {
-    messages("date.range", formatDate(dates.start), formatDate(dates.end))
+  private def rowResult(dates: CashTransactionDates, day: Boolean): String = {
+    messages("date.range",
+      formatDate(dates.start, day),
+      formatDate(dates.end, day))
   }
 
-  private def rowResultWithoutFullStop(dates: CashTransactionDates): String = {
-    messages("date.range", formatDate(dates.start), formatDate(dates.end)).replace(period, emptyString)
+  private def rowResultWithoutFullStop(dates: CashTransactionDates, day: Boolean): String = {
+    messages("date.range",
+      formatDate(dates.start, day),
+      formatDate(dates.end, day)).replace(period, emptyString)
   }
 
-  def formatDate(date: LocalDate)(implicit messages: Messages): String =
-    s"${dateAsDay(date)} ${dateAsMonth(date)} ${date.getYear}"
+  def formatDate(date: LocalDate, day: Boolean)(implicit messages: Messages): String =
```

Comment:
formatDate, dateAsDay and dateAsMonth, can be private methods

Reply:
This breaks all the unit tests for results page summary, whilst I agree that getting everything to be private where possible is a good thing - unless we're deliberately wanting to use it else where - I can address this in 4900 if it's a major consideration but for the time being I think we just leave the functionality as is - especially as I want to limit changes in this area to just reflect the ticket.